docs/src/main/mdoc/migration.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: home
+title:  "scala 3 migration"
+section: "Home"
+position: 7
+---
+
+## Scala 3 Migraion Guide
```

Comment:
Thanks for writing this up! I'm not sure this is a Scala 3 migration guide, since we still support Scala 2.13 as well. I think it's more accurately 0.17 -> 0.18 migration guide, what do you think?

Also I think the notes here are based on the PR and a little out of date. The latest is in https://github.com/typelevel/spire/releases/tag/v0.18.0-M1.

Reply:
Yes, you are right. This is a bit misleading. 

> Also I think the notes here are based on the PR and a little out of date. The latest is in https://github.com/typelevel/spire/releases/tag/v0.18.0-M1.

Thanks, I will update according to the release!

I will re-write what you pointed out soon!